.circleci/config.yml

```diff
@@ -22,21 +33,22 @@ jobs:
             . venv/bin/activate
             pip install --upgrade pip
             pip install --upgrade setuptools
-            cd ml-agents-envs && pip install -e .
-            cd ../ml-agents && pip install -e .
-            pip install pre-commit pytest-cov==2.6.1
-            cd ../gym-unity && pip install -e .
+            pushd ml-agents-envs && pip install -e . && popd
+            pushd ml-agents && pip install -e . && popd
+            pip install -r test_requirements.txt
+            pushd gym-unity && pip install -e . && popd
 
       - save_cache:
           paths:
             - ./venv
-          key: v1-dependencies-{{ checksum "ml-agents/setup.py" }}
+          key: v1-dependencies-py<< parameters.pyversion >>-{{ checksum "ml-agents/setup.py" }}
```

Comment:
Parameterize the cache so that different python versions can get different versions of the packages.

Reply:
Can we add as a comment in the yml file?

Reply:
Done